### PR TITLE
Fixes #4701: Improve accessibility for confetti text

### LINKED ISSUE #4701: [A11YS] - Improve accessibility for confetti text
**Describe the bug**
Accessibility scanner is showing suggestions to improve text scaling and color contrast ratio for confetti text.

**To Reproduce**
1. Turn on accessibility scanner
2. Play exploration
3. Submit correct answer

**Expected behavior**
Accessibility scanner should not show any improvement 

**Screenshot**
image

**Environment**
- Device/emulator being used: Realme GT Master Edition
- Android or SDK version (e.g. Android 5 or SDK 21): Android 12
- App version (you can get this through system app settings or via the admin controls menu in-app): 1.0

### PR DESCRIPTION
**Pull request by vrajdesai78**
## Explanation
Fixes #4701: Improved accessibility for congratulations text by changing height to wrap_content and improved contrast ratio by changing color in color pallet. 

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## Screenshot 

Before         |  After
:-------------------------:|:-------------------------:
image | image

### CODE REVIEW

File: app/src/main/res/layout-land/question_player_fragment.xml [hunk 1]
```diff
@@ -190,4 +191,4 @@
         android:layout_height="124dp" />
     </FrameLayout>
   </FrameLayout>
-</layout>
+</layout>
```
Comment (rt4914) [inline app/src/main/res/layout-land/question_player_fragment.xml]: EOF is missing, add one blank line at the end.

File: app/src/main/res/layout-sw600dp-land/question_player_fragment.xml [hunk 2]
```diff
@@ -193,4 +194,4 @@
         android:layout_height="132dp" />
     </FrameLayout>
   </FrameLayout>
-</layout>
+</layout>
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/question_player_fragment.xml]: EOF is missing, add one blank line at the end.

File: app/src/main/res/layout-sw600dp-port/question_player_fragment.xml [hunk 3]
```diff
@@ -194,4 +195,4 @@
         android:layout_height="132dp" />
     </FrameLayout>
   </FrameLayout>
-</layout>
+</layout>
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-port/question_player_fragment.xml]: EOF is missing, add one blank line at the end.

File: app/src/main/res/layout/question_player_fragment.xml [hunk 4]
```diff
@@ -193,4 +194,4 @@
         android:layout_height="124dp" />
     </FrameLayout>
   </FrameLayout>
-</layout>
+</layout>
```
Comment (rt4914) [inline app/src/main/res/layout/question_player_fragment.xml]: EOF is missing, add one blank line at the end.

File: app/src/main/res/layout/state_fragment.xml [hunk 5]
```diff
@@ -180,4 +181,4 @@
         android:layout_height="124dp" />
     </FrameLayout>
   </FrameLayout>
-</layout>
+</layout>
```
Comment (rt4914) [inline app/src/main/res/layout/state_fragment.xml]: EOF is missing, add one blank line at the end.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@vrajdesai78 Firstly you cannot directly use `component_color_shared_progress_bar_stroke_color`, its not a progress bar its a congratulations text view.

So you will need to update value of `component_color_shared_correct_text_color ` in colors file.

Secondly, please share before and after screenshot side-by-side to clearly show the difference.

File: app/src/main/res/layout-land/question_player_fragment.xml (hunk 1 above)
Comment (vrajdesai78) [reply app/src/main/res/layout-land/question_player_fragment.xml]: added

File: app/src/main/res/layout-sw600dp-land/question_player_fragment.xml (hunk 2 above)
Comment (vrajdesai78) [reply app/src/main/res/layout-sw600dp-land/question_player_fragment.xml]: added

File: app/src/main/res/layout-sw600dp-port/question_player_fragment.xml (hunk 3 above)
Comment (vrajdesai78) [reply app/src/main/res/layout-sw600dp-port/question_player_fragment.xml]: added

File: app/src/main/res/layout/question_player_fragment.xml (hunk 4 above)
Comment (vrajdesai78) [reply app/src/main/res/layout/question_player_fragment.xml]: added

File: app/src/main/res/layout/state_fragment.xml (hunk 5 above)
Comment (vrajdesai78) [reply app/src/main/res/layout/state_fragment.xml]: added

File: app/src/main/res/layout-land/question_player_fragment.xml (hunk 1 above)
Comment (vrajdesai78) [reply app/src/main/res/layout-land/question_player_fragment.xml]: > @vrajdesai78 Firstly you cannot directly use `component_color_shared_progress_bar_stroke_color`, its not a progress bar its a congratulations text view.
> 
> So you will need to update value of `component_color_shared_correct_text_color ` in colors file.
> 
> Secondly, please share before and after screenshot side-by-side to clearly show the difference.

Added screenshot to compare before and after, also updated color_pallete.xml

Comment (vrajdesai78) [reply app/src/main/res/layout-land/question_player_fragment.xml]: > @vrajdesai78 Firstly you cannot directly use `component_color_shared_progress_bar_stroke_color`, its not a progress bar its a congratulations text view.
> 
> So you will need to update value of `component_color_shared_correct_text_color ` in colors file.
> 
> Secondly, please share before and after screenshot side-by-side to clearly show the difference.

Updated description and made changes color_pallete.xml

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

**Pull request merged**